sentry/values.yaml

```diff
@@ -192,6 +192,7 @@ slack: {}
 #   clientId:
 #   clientSecret:
 #   verificationToken:
+#   legacyApp:
```

Comment:
Should we put another comment to reference this issue ?
https://github.com/getsentry/sentry/pull/19446

Reply:
It's certainly better to be able to reference the issue.